packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -55,24 +58,28 @@ export function startCombiner(config: CombinerConfig, kit?: ContractKit) {
     )
   )
 
-  const pnpSign = new Controller(
-    new PnpSignAction(config.phoneNumberPrivacy, new PnpSignIO(config.phoneNumberPrivacy, kit))
-  )
-  app.post(CombinerEndpoint.PNP_SIGN, (req, res) =>
-    meterResponse(pnpSign.handle.bind(pnpSign), req, res, CombinerEndpoint.PNP_SIGN, config)
-  )
-
   const pnpQuota = new Controller(
     new PnpQuotaAction(
       config.phoneNumberPrivacy,
-      new CombinerThresholdStateService(config.phoneNumberPrivacy),
+      combinerThresholdStateService,
       new PnpQuotaIO(config.phoneNumberPrivacy, kit)
     )
   )
   app.get(CombinerEndpoint.PNP_QUOTA, (req, res) =>
```

Comment:
Just noticing I think this should be POST right?

Reply:
I was seeing errors when get requests were supplied bodies

Reply:
yeah agreed, and for consistency with the signer quota endpoint